src/libvirtApi/common.ts

```diff
@@ -166,35 +180,33 @@ async function doUsagePolling(name, connectionName, objPath) {
     const flags = Enum.VIR_DOMAIN_STATS_BALLOON | Enum.VIR_DOMAIN_STATS_VCPU | Enum.VIR_DOMAIN_STATS_BLOCK | Enum.VIR_DOMAIN_STATS_STATE;
 
     try {
-        const [info] = await call(connectionName, objPath, "org.libvirt.Domain", "GetStats", [flags, 0], { timeout: 5000, type: "uu" });
+        const [info] = await call<[DBusProps]>(connectionName, objPath, "org.libvirt.Domain", "GetStats", [flags, 0], { timeout: 5000, type: "uu" });
         if (Object.getOwnPropertyNames(info).length > 0) {
-            const props = { name, connectionName, id: objPath };
+            const props: Partial<VM> = { name, connectionName, id: objPath };
             let avgvCpuTime = 0;
 
             if ("balloon.rss" in info)
-                props.rssMemory = info["balloon.rss"].v.v;
-            else if ("state.state" in info && info["state.state"].v.v == Enum.VIR_DOMAIN_SHUTOFF)
+                props.rssMemory = (info["balloon.rss"].v as DBusVariant).v as number;
```

Comment:
Ideally we shouldn't use as `DBusVariant` if the type `DBusType` doesn't have it (or DBusVariant doesn't have it). Also this kinda goes around any type assertions 

So could do something like:
```
export type DBusType = DBusVariant | string | Uint8Array | number | boolean | DBusType[] | { t: string; v: DBusType };
```